code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -365,7 +377,7 @@
 /obj/item/storage/backpack/syndicate_medic_satchel
```

Comment:
please refactor to be a child of `/obj/item/storage/backpack/satchel/syndie`

Reply:
did this thing